src/lib/components/constrainedtext/Constrainedtext.component.js

```diff
@@ -1,14 +1,14 @@
 //@flow
-import React from 'react';
+import React, {useEffect, useRef, useState} from 'react';
 import type { Node } from 'react';
 import styled from 'styled-components';
 import Tooltip from '../tooltip/Tooltip.component.js';
 import type { Props as TooltipProps } from '../tooltip/Tooltip.component.js';
 type Props = {
   text: string,
   tooltipStyle?: $PropertyType<TooltipProps, 'overlayStyle'>,
-  tooltipPlacement?: $PropertyType<TooltipProps, 'placement'>,
-  lineClamp?: number,
+    tooltipPlacement?: $PropertyType<TooltipProps, 'placement'>,
```

Comment:
I think your editor is not configured to format the code using prettier, we can check that together so that the code style remains homogenous

Reply:
I just installed Prettier on my IDE. Thanks!